src/rollback_to_stable/rts_api.c

```diff
@@ -345,4 +351,32 @@ __wt_rollback_to_stable_init(WT_CONNECTION_IMPL *conn)
 
     /* Setup variables. */
     conn->rts->dryrun = false;
+
+    /* Setup worker threads at connection level. */
+    if ((ret = __wt_config_gets(session, cfg, "rollback_to_stable.threads", &cval)) == 0)
+        conn->rts->cfg_threads_num = (uint32_t)cval.val;
+    WT_RET_NOTFOUND_OK(ret);
+
+    return (0);
+}
+
+/*
+ * __wt_rollback_to_stable_reconfig --
+ *     Re-configure the RTS worker threads.
+ */
+int
+__wt_rollback_to_stable_reconfig(WT_SESSION_IMPL *session, const char **cfg)
+{
+    WT_CONFIG_ITEM cval;
+    WT_CONNECTION_IMPL *conn;
+    WT_DECL_RET;
+
+    conn = S2C(session);
+
+    /* Re-configure the RTS worker threads at connection level. */
+    if ((ret = __wt_config_gets(session, cfg, "rollback_to_stable.threads", &cval)) == 0)
+        conn->rts->cfg_threads_num = (uint32_t)cval.val;
```

Comment:
This reconfiguration is a consideration for computing for `conn_session_size`. Both LSM and eviction have a `threads_min` and a `threads_max` value and use the `max` value for allocating session entries. This change should either provide the same min-max configurations, or use the suggested macro definition to allocate the maximum allowed at the start `WT_RTS_THREADS_MAX` so that all reconfigurations can be assured of a session slot.